lib/class-wp-rest-menus-controller.php

```diff
@@ -348,18 +348,11 @@ public function create_item( $request ) {
 		$term = wp_update_nav_menu_object( 0, wp_slash( (array) $prepared_term ) );
 
 		if ( is_wp_error( $term ) ) {
-			/*
-			 * If we're going to inform the client that the term already exists,
-			 * give them the identifier for future use.
-			 */
-			$term_id = $term->get_error_data( 'term_exists' );
-			if ( $term_id ) {
-				$existing_term = get_term( $term_id, $this->taxonomy );
-				$term->add_data( $existing_term->term_id, 'term_exists' );
```

Comment:
The `wp_update_nav_menu_object` returns custom error message with `menu_exists` code. We have no additional data, like Menu ID.